source/containers/document-pii-detection/parsers/image_analysis/general_ocr/postprocess/__init__.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import copy
+
+__all__ = ['build_post_process']
+
+
+def build_post_process(config, global_config=None):
+    from .db_postprocess import DBPostProcess
+    from .rec_postprocess import CTCLabelDecode, AttnLabelDecode
+    from .cls_postprocess import ClsPostProcess
+
+    support_dict = [
+        'DBPostProcess', 'CTCLabelDecode', 'AttnLabelDecode', 'ClsPostProcess'
+    ]
+
+    config = copy.deepcopy(config)
+    module_name = config.pop('name')
+    if global_config is not None:
+        config.update(global_config)
+    assert module_name in support_dict, Exception(
+        'post process only support {}'.format(support_dict))
+    module_class = eval(module_name)(**config)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

If you do not have absolute control over the contents of the string passed to the `eval()` function, they might be maliciously used to execute an arbitrary function such as `__import__(&#39;os&#39;).system(&#34;rm -rf $HOME&#34;)`. If the string is expected to only contain literals, consider using the `ast.literal_eval()` method to safely evaluate strings containing Python values from untrusted sources.

[Learn more](https://docs.python.org/3/faq/programming.html#how-do-i-use-strings-to-call-functions-methods)

Reply:
Resolve by replacing the eval function with creating specific Python classes.